core/sql/langman/LmRoutineJavaObj.cpp

```diff
@@ -192,6 +192,7 @@ LmResult LmRoutineJavaObj::invokeRoutineMethod(
           (*emitRowPtr_)(NULL,0,&qs);
         }
     }
+  jni->DeleteLocalRef(jniResult);
```

Comment:
I think, unless  jniResult object reference is retained in the java space or it is made as global reference in JNI, JVM wouldn't guarantee the correctness of jniResult object across two different invocations of the JNI method.